common.sh

```diff
@@ -39,6 +39,11 @@ source $CONFIG
 export OPENSHIFT_RELEASE_IMAGE="${OPENSHIFT_RELEASE_IMAGE:-registry.svc.ci.openshift.org/ocp/release:4.2}"
 export OPENSHIFT_INSTALL_PATH="$GOPATH/src/github.com/openshift/installer"
 
+if env | grep -q "_LOCAL_IMAGE=" ; then
+    # We need a custome installer (allows http image pulls for local images)
```

Comment:
Any more info on this - is it a configuration that's turned off in the release-payload installer build?

Reply:
Its needed because in 04_setup_ironic.sh we are adding a file into the filesystem 

`    echo -e "[registries.insecure]\nregistries = ['192.168.111.1:5000']" > $OPENSHIFT_INSTALL_PATH/data/data/bootstrap/baremetal/files/etc/containers/registries.conf `

we need to build a custom installer including this file

Reply:
Ah I see, thanks - I saw the `99_local-registry.yaml.optional` and assumed we could do all the local-registry config via an asset, but I guess we need registries.conf before the MCO runs.

Modifying the installer checkout seems fine, but one alternative would be to generate the ignition-configs via the installer, then inject the registries.conf via ignition - we did that early in dev-scripts IIRC for some other things.

Reply:
> Modifying the installer checkout seems fine, but one alternative would be to generate the ignition-configs via the installer, then inject the registries.conf via ignition - we did that early in dev-scripts IIRC for some other things.

I'll look into doing this as an alternative

Reply:
> I'll look into doing this as an alternative

Not necessarily saying that we should, just wanted to mention it as a possible alternative :+1: